### ISSUE TITLE
Way to disable syntax error highlighting in insert mode?

### ISSUE THREAD

**Issue by xiaochuanyu** [numirias/semshi#117]
Currently, syntax errors are highlighted after some delay when you are stop typing in insert mode as seen here:
![image](https://user-images.githubusercontent.com/1718745/149189025-210cb1ff-53d2-4b79-8d63-e36b5a09f0f5.png)
Is there a way to disable highlighting syntax errors while insert mode?
E.g. I expect the following to be shown as long as I'm in insert mode:
![image](https://user-images.githubusercontent.com/1718745/149189398-20d03949-e03d-4c1c-9c1c-9021d18e5964.png)

**Comment by smprather** [comment]
Yes. I am here looking for the same. Even though I have error marker delay set to the default of 1.5s, it is not honored while typing in insert mode. So the error column is quickly bouncing between on/off while I type making everything jerk left and right as I type. I also wouldn't mind an option to always have the error-marker-column visible. Whatever it takes to stop the bouncing while I'm typing. Thanks!

**Comment by smprather** [comment]
I was was going to try to use autocmd to turn the error marker on/off when entering/leaving insert mode, but the variable does not appear to be "live" after loading init.vim. So a possible solution to this issue is to make g:semshi#error_sign responsive to changes after init.vim is loaded.
